From 315e2a0365efa7bbc267542fe2b9a85e0e13eef0 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Thu, 2 Jan 2020 17:32:33 +0200
Subject: [PATCH 41/55] ARM: dts: imx7d: cl-som-imx7: disable unsupported
 operating point

Redefine the 'operating-points' array to include only operating points
supported by Cl-SOM-iMX7.

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 arch/arm/boot/dts/imx7d-cl-som-imx7.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-cl-som-imx7.dts b/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
index 10f35617ba45..987f43067291 100644
--- a/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
+++ b/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
@@ -128,6 +128,16 @@
 
 &cpu0 {
 	arm-supply = <&sw1a_reg>;
+	operating-points = <
+		/* KHz	uV */
+		996000	1075000
+		792000	975000
+	>;
+	clock-frequency = <996000000>;
+};
+
+&cpu1 {
+	clock-frequency = <996000000>;
 };
 
 &ecspi1 {
-- 
2.20.1

